code/modules/surgery/organs/internal/ears/_ears.dm

```diff
@@ -149,4 +149,4 @@
 	. = ..()
 	if(. & EMP_PROTECT_SELF)
 		return
-	apply_organ_damage(40/severity)
+	apply_organ_damage(10 * severity)
```

Comment:
But why 

https://github.com/tgstation/tgstation/blob/bbdbe55c69775aa6d558329a309170bc3ee14a1c/code/__DEFINES/combat.dm#L259-L260

Reply:
holy fucking shit

Reply:
this sucks i hate it why is it like this